packages/slate-renderer/src/components/Media/Media.scss

```
@@ -3,19 +3,18 @@
 
 .prezly-slate {
     &-media {
-        &--image {
+        &--image,
+        &--video {
             display: block;
             max-width: 100%;
             height: auto;
             margin: 0 auto;
+        }
```

Since it applies to all `.prezly-slate-media`, we can skip the modifiers

```suggestion
        display: block;
        max-width: 100%;
        height: auto;
        margin: 0 auto;
```

---

Sorry, merged the PR before your comment. Will apply this fix separately.